src/components/Questionnaire/QuestionnaireTableStyle.tsx

```diff
@@ -0,0 +1,35 @@
+import {styled} from "@mui/material/styles";
```

Comment:
vllt ganze File ins Theme packen. Dann können einzelene Themes es überschreiben, aber ein default ist gesetzt. So wie bei button "contained" in theme.ts

Reply:
Wird an einen SHK später weitergegeben